bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt

```diff
@@ -51,7 +51,7 @@ class ModelImporter(private val root: INode, private val continueOnError: Boolea
     // updated to the constructor with two arguments.
     constructor(root: INode) : this(root, false)
 
-    private fun doAndPotentiallyContinueOnErrors(block: () -> Unit) {
+    private inline fun doAndPotentiallyContinueOnErrors(block: () -> Unit) {
```

Comment:
Maybe add a comment for why this was inlined?

Reply:
I'm not sure if this needs further explanation, as it is very similar to the [prime example for inline functions](https://kotlinlang.org/docs/inline-functions.html).

Reply:
We probably have a good reason why we go the special case here for inlining it. It would be bad, if that reasoning gets lost in time and the next one touching the code doesn't know about it anymore.

Reply:
I agree that documenting the reasons is unnecessary, because it would just repeat the Kotlin documentation for when to use this keyword. You can expect an experienced Kotlin developer to know this.

Reply:
I understand that documentation is tedious - especially if it is self written code and "obvious". but please consider that even our current teams is not composed of only kotlin experts.

Reply:
Maybe I phrased it wrong. It doesn't require any experience to look this keyword up in the Kotlin docs, in case someone is unfamiliar with it. It's just a normal language feature used in it's intended way. It's not about being tedious work to document it, it just noise. Bad document is worse than no documentation.

Reply:
We can move a bit more of rational to the commit message maybe to avoid the comment. There's a history of a report that this might have slowed down execution. This information would be lost in time if it's not written down somewhere permanently and well-discoverable from the code base.

Reply:
The claims of that report has never been verified and I don't want to add possibly false information. 
The rationale was to improve performance no matter if the difference is noticeable to a user. The commit prefix perf conveys sufficient information. If there is an unexperienced dev, who is curious why this improves performance, they can read the documentation for the keyword. We shouldn't overthink this.